### ISSUE TITLE
can not import coda

### ISSUE THREAD

**Issue by htyeim** [stcorp/coda#15]
I was installed codasetup-win64-2.17.2.exe and python setup.py install.
But can not import the coda package
the error message as followed:
```

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "D:\CODA\python\coda\__init__.py", line 22, in <module>
    from .codapython import *
  File "D:\CODA\python\coda\codapython.py", line 22, in <module>
    from .codac import *
  File "D:\CODA\python\coda\codac.py", line 26, in <module>
    _codac = swig_import_helper()
  File "D:\CODA\python\coda\codac.py", line 22, in swig_import_helper
    _mod = imp.load_module('_codac', fp, pathname, description)
  File "F:\User\iht\Program Files\Anaconda3\lib\imp.py", line 242, in load_module
    return load_dynamic(name, filename, file)
  File "F:\User\iht\Program Files\Anaconda3\lib\imp.py", line 342, in load_dynamic
    return _load(spec)
```
OS: windows 10
python: anaconda python 3.5

Is this package can not use in python 3?

**Comment by svniemeijer** [comment]
There should be more information from the traceback, most importantly the error message itself.
Could you please provide that?

Also note that the python interface that comes with the CODA binary installer only works with Python 2.7 (as described in the documentation). If you want to use it with Python 3, you would have to build CODA from source yourself (using, for instance, the CMake build).

**Comment by htyeim** [comment]
Thank you for reply!
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "D:\CODA\python\coda\__init__.py", line 22, in <module>
    from .codapython import *
  File "D:\CODA\python\coda\codapython.py", line 22, in <module>
    from .codac import *
  File "D:\CODA\python\coda\codac.py", line 26, in <module>
    _codac = swig_import_helper()
  File "D:\CODA\python\coda\codac.py", line 22, in swig_import_helper
    _mod = imp.load_module('_codac', fp, pathname, description)
  File "F:\User\iht\Program Files\Anaconda3\lib\imp.py", line 242, in load_module
    return load_dynamic(name, filename, file)
  File "F:\User\iht\Program Files\Anaconda3\lib\imp.py", line 342, in load_dynamic
    return _load(spec)
ImportError: DLL load failed: 找不到指定的模块。
```
or The specified module could not be found
.
_mod = imp.load_module('_codac', fp, pathname, description)
I think the problem was the module "_codac" can not found.
Or maybe the python version.....

I am try to find some python package which can read RINEX file. But maybe calculation related to this type of file need some program language like C Fortran. 
This project is great, and  the feature described in http://www.stcorp.nl/coda/doc/codadef/codadef-rinex.html just what I search for. But it may be hard to use in python. If this project could provide something like pip install coda or conda install coda, this project could become popular.
Thank you for write this great project!~

**Comment by svniemeijer** [comment]
You should be able to use the CODA binary package with Python 2.7, so you migth try it with Python 2.7.

I agree that a conda version would be usefull, which is why I created #14 when I saw your CODA registration remark.

**Comment by htyeim** [comment]
:) Thank you
I think many people might feel depressed when they face too many scientific data formats. This project can be very useful.
I can use C++, Fortran and C# to read RINEX o, n, g file, but recently, I am using python 3 after I found the great matplotlib and pandas package.  I just don't want to write this type of program, so I search for some exist python package. I found gpstk and this package, but both don't support python 3, and can't install conveniently...
